src/pages/caseDetail/caseDetailView.js

```diff
@@ -51,18 +52,39 @@ const options = (classes) => ({
   ),
 });
 
+const PropertyItem = ({
+  classes, label, dataField, defaultValue, linkUrl,
+}) => (
+  <Grid item xs={12}>
+    <Grid container spacing={4}>
+      <Grid item xs={4}>
+        <span className={classes.title}>{label}</span>
+      </Grid>
+      <Grid item xs={8} className={classes.content}>
+        { dataField != null ? (
+          linkUrl ? <Link to={linkUrl} className={classes.link}>{dataField}</Link>
+            : dataField
+        ) : defaultValue }
+      </Grid>
+    </Grid>
+  </Grid>
+);
+
 const CaseDetail = ({ classes, data }) => {
   const stat = {
-    numberOfTrials: 1,
-    numberOfCases: 1,
-    numberOfFiles: data.filesOfCase.length,
+    numberOfPrograms: 1,
```

Comment:
Can you think of a better way where we can use global header elements and assign accordingly?

Reply:
Any suggestions?